cookbooks/aws-parallelcluster-install/resources/fabric_manager/partial/_fabric_manager_install_debian.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+#
+# Copyright:: 2013-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :install_package do
+  # For ubuntu, CINC17 apt-package resources need full versions for `version`
+  execute "install_fabricmanager_for_ubuntu" do
+    command "apt -y install #{node['cluster']['nvidia']['fabricmanager']['package']}=#{node['cluster']['nvidia']['fabricmanager']['version']} "\
```

Comment:
in scenarios like this one, can't the resource own and encapsulate the default package name and let it to be customized via a property if need be, this will allow to reduce un-encapsulated direct access to node values

Reply:
I agree that the resource can be refactored and the arguments can be passed as a properties, but it is out of scope for this PR, I will add a NTH task to track this.